driver/pparse.ml

```diff
@@ -184,6 +184,7 @@ let file_aux ~tool_name inputfile (type a) parse_fun invariant_fun
         seek_in ic 0;
         let lexbuf = Lexing.from_channel ic in
         Location.init lexbuf inputfile;
+        Location.input_lexbuf := Some lexbuf;
```

Comment:
I asked myself about the impact of this change on external users, but I thought about it more and I don't see anything wrong with it, in fact it was probably a bug not to have it there -- in the work we did on expect-style tests we have a similar setting that was necessary for us.